docs/products/clickhouse/concepts/indexing.rst

```diff
@@ -0,0 +1,21 @@
+Indexing and data processing
+=============================
+
+In this article we look at how ClickHouse works with data and how this differs from other database management systems.
+In particular we'll touch on such topics as **data indexing** and **a vector computation engine**.
+
+Understanding ClickHouse approach to data processing will help you learn why ClickHouse is perfect solution for some use cases and an awful choice for others.
+
```

Comment:
I would put a subtitle here like `Columnar Database` and expand a bit.

You might compare it with the traditional databases storing a row at the time.

Reply:
We have a separate article on it. I can add more, but then I'm duplicating it... Added a clear reference to the separate article. Do you think this makes it more clear?

if not, I can indeed add some more sentences about columnar db

Reply:
I think the link addresses what is needed here